.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  release:
+    types: [created]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        scala_version: &scala_versions
+          - 2.12.12
+          - 2.13.4
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with: &cache_options
+          path: |
+            ~/.ivy2/cache
+            ~/.sbt
+            ~/.coursier
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/*.sbt') }}-${{ matrix.scala_version }}
```

Comment:
should the other directories be part of the hash key too?

Reply:
what other directories?

Reply:
Ah, I misread that. I thought you were using the hash of the files in the .sbt dir, but it looks like you're using the hash of all files with the .sbt extension. I'm not as familiar here, is that all we need for the cache key? 

Aside, re my original question: it doesn't even make sense for the cached directories to be part of the key cause we won't have their contents until after we do the lookup :upside_down_face: